playbook.md

```diff
@@ -119,31 +120,20 @@ of wisdom, but we don’t follow the industry of methodologies, training and
 certifications that has grown up around it. We think agile is something you
 learn to be, not something that you learn to do.
 
-Our teams frequently experiment with different tools and techniques so we can
-improve. This applies not only to development work, but to everything we do at
-dxw, from user research to business operations.
+We recognise the importance of compliance and governance, so we build those activities into each step.
 
-We recognise the importance of planning, compliance and governance, so we work
-those activities into the process, every step of the way.
+#### Work at a sustainable pace
 
-#### Sustainable pace
-
-We work at a sustainable pace. We estimate work and schedule it conservatively,
+At dxw, we work at a sustainable pace. We estimate work and schedule it conservatively,
 and set realistic expectations with clients about the pace of delivery.
 
 This doesn't mean that we don't work hard. Being productive is a vital part of
-maintaining the trust that our clients give us, and it's important not to let
+maintaining the trust that our clients give us, and it’s important not to let
 them down.
 
-When taking time off during client work, we discuss how it will impact the
-schedule with other team members.
-
-Sending communication outside of working hours may create an unintended sense of
-urgency with the recipients of your message. Try to avoid creating that urgency
-when possible. Unless actually urgent, you may ignore messages which you receive
-outside of working hours and handle them once you are back at work.
```

Comment:
How come we're removing this? I think it's an important callout to our culture of sustainability, and how it might impact team and client.

Reply:
It is important stuff. But it's detailed guidance. We have a section later in the playbook about time and flexible working. We should integrate this material into those sections.

Reply:
Moved the material into the Time section.